app/views/admin/loans/show.html.slim

```diff
@@ -1,37 +1,73 @@
 - content_for(:title, @loan.name)
 
 - content_for :styles
-  = stylesheet_link_tag "admin/loans/loan", :media => "all"
+  = stylesheet_link_tag "admin/forms/forms", :media => "all"
+  = stylesheet_link_tag "admin/loans/tabs", :media => "all"
   = stylesheet_link_tag "admin/loans/timeline", :media => "all"
 
-div.loan-content
-  // Nav tabs
-  ul class="nav nav-tabs" role="tablist"
-    li role="presentation"
-      a href="#details" aria-controls="details" role="tab" data-toggle="tab"
-        ="Details"
-    li role="presentation"
-      a href="#criteria" aria-controls="criteria" role="tab" data-toggle="tab"
-        = "Criteria"
-    li role="presentation" class="active"
-      a href="#timeline" aria-controls="timeline" role="tab" data-toggle="tab"
-        = "Timeline"
-    li role="presentation"
-      a href="#calendar" aria-controls="calendar" role="tab" data-toggle="tab"
-        = "Calendar"
-    li role="presentation"
-      a href="#print" aria-controls="print" role="tab" data-toggle="tab"
-        = "Print"
-    li role="presentation"
-      a href="#more" aria-controls="more" role="tab" data-toggle="tab"
-        = "More"
-
-  // Tab panes
-  div class="tab-content"
-    div role="tabpanel" class="tab-pane" id="details" ...
-    div role="tabpanel" class="tab-pane" id="criteria" ...
-    div role="tabpanel" class="tab-pane active" id="timeline"
-      = render "admin/loans/timeline/timeline", loan: @loan
-    div role="tabpanel" class="tab-pane" id="calendar" ...
-    div role="tabpanel" class="tab-pane" id="print" ...
-    div role="tabpanel" class="tab-pane" id="more" ...
+- content_for :scripts
+  = javascript_include_tag "admin/forms/forms"
+
+section.loan.details class=(@loan.valid? ? 'show-view' : 'edit-view')
+
+  header
+    .show-actions
+      - if @loan.valid?
+        a.edit-action.view-element href="#"
+          i.fa.fa-pencil.fa-large>
+          = t(:edit)
+        a.show-action.form-element href="#" = t(:cancel)
+      - else
+        a href=admin_organization_path(@loan) = t(:cancel)
+      = link_to [:admin, @loan], method: :delete,
+        data: { confirm: t('loan.confirm_deletion') }
+        i.fa.fa-trash.fa-large>
+        = t(:delete)
+    .clearfix
+
+  div.loan-content
+    // Nav tabs
```

Comment:
The nav tabs are a good candidate to put into a partial, otherwise everything looks great!